portfolio/src/main/webapp/script.js

```diff
@@ -307,14 +307,94 @@ function createBlobElement(blob){
 
 let map;
 
-function initMap() {
-  map = new google.maps.Map(document.getElementById("map"), {
-    center: {
-      lat: 45.5351,
-      lng: 25.2181
-    },
-    zoom: 11,
-    mapTypeId: "satellite"
+/* Editable marker that displays when a user clicks in the map. */
+let editMarker;
+
+/** Creates a map that allows users to add markers. */
+function createMap() {
+  map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 38.5949, lng: -94.8923}, zoom: 4});
```

Comment:
what is in Kansas? :-)

Reply:
The place where Dorothy adventure from Wizard of Oz started. Haha, it was initially Piatra Craiului Mountains. I will change it to something more relevant. Check out the map from here: 
[(https://teoreu-step-2020.ew.r.appspot.com/)]

Reply:
I invite you to add places as well.